front/app/containers/ProjectsShowPage/timeline/PhaseDescriptions.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
To implement the tab list from above, the phase descriptions needed to appear as one element below the timeline. Before, they were mixed in the same component.